README.md

```diff
@@ -137,6 +137,70 @@ export default withOnyx({
 
 It is preferable to use the HOC over `Onyx.connect()` in React code as `withOnyx()` will delay the rendering of the wrapped component until all keys have been accessed and made available.
 
+## Collections
+
+Collections allow keys with similar value types to be subscribed together by subscribing to the collection key. To define one, it must be included in the `ONYXKEYS.COLLECTION` object and it must be suffixed with an underscore. Member keys should use a unique identifier or index after the collection key prefix (e.g. `chat_42`).
+
+```javascript
+const ONYXKEYS = {
+    COLLECTION: {
+        CHAT: 'chat_',
+    },
+};
+```
+
+### Setting Collection Values
+
+To save a new collection key we can either do:
+
+```js
+Onyx.merge(`${ONYXKEYS.COLLECTION.CHAT}${chat1.id}`, chat1);
+```
+
+or we can set many at once with `mergeCollection()`:
+
+```js
+Onyx.mergeCollection(ONYXKEYS.COLLECTION.CHAT, {
+    [`${ONYXKEYS.COLLECTION.CHAT}${chat1.id}`]: chat1,
+    [`${ONYXKEYS.COLLECTION.CHAT}${chat2.id}`]: chat2,
+    [`${ONYXKEYS.COLLECTION.CHAT}${chat3.id}`]: chat3,
+});
+```
+
+### Subscribing to Collections
+
+There are several ways to subscribe to these keys:
+
+```javascript
+withOnyx({
+    allChats: {key: ONYXKEYS.COLLECTION.CHAT},
+})(MyComponent);
+```
+
+This will return the initial collection as an object of collection member key/values. Changes to the individual member keys will modify the entire object and new props will be passed with each individual key update.
+
+```js
+Onyx.connect({key: ONYXKEYS.COLLECTION.CHAT}, callback: (memberValue, memberKey) => {...}});
+```
+
+This will fire the callback `n` times depending on how many collection member keys have been stored. Changes to those keys after the initial callbacks fire will occur when each individual key is updated.
```

Comment:
Where is `n` defined? I don't see it in the code above. Is it once per member key? Either way, might be good to clarify here. (Coming as someone with no context on this whatsoever.)